packages/seedless-onboarding-controller/src/SeedlessOnboardingController.ts

```
@@ -500,6 +571,8 @@ export class SeedlessOnboardingController<EncryptionKey> extends BaseController<
     return await this.#withControllerLock(async () => {
       await this.#unlockVaultAndGetBackupEncKey(password);
       this.#setUnlocked();
+      // revoke and recyle refresh token after unlock to keep refresh token fresh, avoid malicious use of leaked refresh token
+      await this.revokeRefreshToken(password);
```

what happens if we fail to revoke refresh token, will it restrict user to unlock wallet or it just fails silently?

---

@himanshuchawla009 currently this one is after wallet is unlocked so it will fail silently
do you think we should keep it like this